apps/iOS-WKWebView-sample/WkWebView DemoUITests/WkWebView_DemoUITests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  WkWebView_DemoUITests.swift
+//  WkWebView DemoUITests
+//
+//
+
+import XCTest
+
+class WkWebView_DemoUITests: XCTestCase {
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+    }
+
+
+    func joinMeetingTest() throws {
```

Comment:
I think it should start with `test`

Reply:
See some of following [code](https://github.com/xamarin-automation-service/xcuitest-pop-example/blob/master/ToDoListUITests/ToDoListUITests.swift#L11)